frontend/src/lib/services/Libre311/Libre311.ts

```diff
@@ -417,8 +417,20 @@ export class Libre311ServiceImpl implements Libre311Service {
 		throw Error('Not Implemented');
 	}
 
-	async uploadImage(file: File): Promise<unknown> {
-		throw Error('Not Implemented');
+	async uploadImage(fileData: string): Promise<string> {
+		try {
+			const res = await axios.post("http:://localhost:8080/api/image", {
```

Comment:
You can use the axios instance here
ex: `axiosInstance.post("/image", {...})`

That way if in the future we add any sort of axios interceptors to the instance then they would be applied here too.